### ISSUE TITLE
Error upgrading Statamic to v 3.3

### ISSUE THREAD

**Issue by DrTrills** [justkidding96/statamic-aardvark-seo#120]
I upgraded my local environment to 3.3 and now I get:

```
Facade\Ignition\Exceptions\ViewException
Call to undefined method Statamic\Fields\Value::format()
```
If I delete the content of:
`vendor/withcandour/aardvark-seo/resources/views/tags/head.antlers.html `

The issue goes away, or if I remove `{{ aardvark-seo:head }}` from my template.

Any help would be appreciated. Thanks

**Comment by dave-smyth** [comment]
I get the same. This might be related to @xuneXTW’s PR here:
https://github.com/WithCandour/statamic-aardvark-seo/pull/119/commits/f01655b19ea471979dee3a2d2f7da2b942199413

Hopefully a fix soon.

**Comment by AndrewHaine** [comment]
Hi both,

Thank you for reporting, I've confirmed that this was indeed the same issue fixed in the linked PR. 

A fix has been released in 2.0.22.
